### ISSUE TITLE
Error when loading relationship on entity created with omitted relation property

### ISSUE THREAD

**Issue by NZX-DeSiGN** [revolver-app/vuex-orm#29]
Hi,

First thanks for your awesome plugin and forgive my english please !

If I create an entity and omiting the relationship property, the entity is added to the store without error. But when I try to get the entity with it relation, I'm getting the following error "TypeError: Cannot read property 'map' of undefined". 

I think it come from a lodash method and to solve that issue we must create default fields (as described in the model) not existing on entity creation. This could also resolve the issue https://github.com/revolver-app/vuex-orm/issues/25

Here's an example : 

````js
import Post from './Post'

export default class User extends Model {
  static entity = 'user'

  static fields () {
    return {
      id: this.attr(null),
      username: this.attr('')
      posts: this.hasManyBy(Post, 'posts')
    }
  }
}

const data = {
  id: 1,
  username: 'Test'
}

store.dispatch('entities/user/create', { data })
````
The above code doesn't throw an error, but if I call the getter below, it will throw the error "TypeError: Cannot read property 'map' of undefined"
````js
store.getters['entities/user/query']().with('posts').get()
````
Currently to fix this error the data should contain the empty array property "posts" as following :  
````js
const data = {
  id: 1,
  username: 'Test',
  posts: []
}
````

**Comment by kiaking** [comment]
Hi, thanks for reporting!

Yeas this is related to #25 and by fixing #25, this issue should also go away. Which means I should harry fixing that 😈

**Comment by kiaking** [comment]
Made PR #33 which should solve this issue.

**Comment by kiaking** [comment]
This issue should be fixed with Vuex ORM 0.13.2. If you still have some related problem after updating Vuex ORM to 0.13.2, please feel free to reopen this issue.
